stylesheets/pulsar.scss

```diff
@@ -111,6 +111,7 @@ $fa-css-prefix: 'icon';
 @import 'component.icons';
 @import 'component.input-groups';
 @import 'component.labels';
+@import 'component.lexicon';
```

Comment:
Suspect this would be better at the end of the imports, with the other override/scratch styles, we might need to use the source order to our advantage

Reply:
Will rework that in ticket #641